File: app/src/main/java/pl/poznajapp/model/Story.java
package pl.poznajapp.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by rafal on 22.08.17.
 */

public class Story {

    @SerializedName("id")
    @Expose
    private Integer id;
    @SerializedName("title")
    @Expose
    private String title;
    @SerializedName("description")
    @Expose
    private String description;
    @SerializedName("duration")
    @Expose
    private String duration;
    @SerializedName("story_images")
    @Expose
    private List<Image> storyImages = null;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getDuration() {
        return duration;
    }

    public void setDuration(String duration) {
        this.duration = duration;
    }

    public List<Image> getStoryImages() {
        return storyImages;
    }

    public void setStoryImages(List<Image> storyImages) {
        this.storyImages = storyImages;
    }

}

File: app/src/main/java/pl/poznajapp/model/Story.kt
package pl.poznajapp.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by Rafał Gawlik on 11.08.17.
 */

class Story {

    @SerializedName("id")
    @Expose
    var id: Int? = null
    @SerializedName("title")
    @Expose
    var title: String? = null
    @SerializedName("description")
    @Expose
    var description: String? = null
    @SerializedName("duration")
    @Expose
    var duration: String? = null
    @SerializedName("story_images")
    @Expose
    var storyImages: List<Image>? = null

}
